blocks/api/post.pegjs

```diff
@@ -95,7 +112,7 @@ WP_Block_Balanced
   }
 
 WP_Block_Html
-  = ts:(!WP_Block_Balanced !WP_Block_Void c:Any {
+  = ts:(!WP_Block_Balanced !WP_Block_Void !WP_Block_NextPage c:Any {
```

Comment:
It's up to you. It would be nice if they were both the same. I chose `Tag` because of the exceptional nature of the tag, not knowing originally it would turn into a block. I think that difference is legitimate here since it's clearly not a block in the text.

Either way, could we normalize them both to match each other?